simulator.py

```diff
@@ -146,11 +227,13 @@ def autoplay(self):
             + ".csv"
         )
         with open(fname, "w") as fo:
```

Comment:
I recommend adding `tournament_results/` dir creation here, so that people don't need to do it manually. E.g. importing `from pathlib import Path` and then something like:
```py
fname = Path("tournament_results") / f"{self.world.player_1_name}_vs_{self.world.player_2_name}_at_{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.csv"
fname.parent.mkdir(exist_ok=True)
```